yarn.lock

```diff
@@ -73,6 +73,14 @@
   dependencies:
     "@babel/highlight" "^7.16.7"
 
+"@babel/code-frame@^7.22.13", "@babel/code-frame@^7.23.4":
```

Comment:
confirm plz that u have run `yarn-deduplicate` in your branches? if yes - let's merge

Reply:
Should I add this command as a script in `package.json`? Cause I thought `yarn` already did some dedup with https://classic.yarnpkg.com/en/docs/cli/dedupe

Reply:
`yarn.lock` is updated